### PR TITLE
feat :products, filtering, fix: nav-aside

### PR DESCRIPTION
**Pull request by BelideNikhil**
added=> Products page,
                Filtering section
                mobile responsive products and filter section

### CODE REVIEW

File: src/Components/FilterSection/FilterSection.jsx [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+                        <div>
+                            <i className="fas fa-rupee-sign"></i>9999
+                        </div>
+                    </div>
+                    <input
+                        type="range"
+                        min="99"
+                        max="9999"
+                        step="99"
+                        className="range-track"
```
Comment (pritam-kr) [inline src/Components/FilterSection/FilterSection.jsx:48]: You can also get max price from the product array. instead of doing hard coded

**Review by pritam-kr — APPROVED** [review]
Good Nikhil. everything good to me

- your helper function is pure.
- clean code
-

File: src/Components/FilterSection/FilterSection.jsx [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+                        />
+                        Tail Tidy
+                    </label>
+                    <label>
+                        <input
+                            type="checkbox"
+                            onChange={(e) =>
+                                productDispatchFun({
+                                    type: "CATEGORY",
+                                    payload: { category: "HANDGUARDS", value: e.target.checked },
```
Comment (poornima-alugubelly) [inline src/Components/FilterSection/FilterSection.jsx:189]: you can make a constants file for all these action types

**Review by poornima-alugubelly — APPROVED** [review]

**Comment by poornima-alugubelly** [comment]
Nice naming convention and file structure 👍

File: src/Components/FilterSection/FilterSection.jsx (hunk 2 above)
Comment (BelideNikhil) [reply src/Components/FilterSection/FilterSection.jsx:189]: made the changes, thanks for the feedback!

**Pull request merged**